test.sh

```diff
@@ -62,6 +62,11 @@ for version in "${meteor_versions[@]}"; do
 		node_version='8.17.0'
 	fi
 
+	# Versions 2.3 need Node 14.17.1
+	if [[ "${version}" == 2.3* ]]; then
+		node_version='14.17.1'
+	fi
```

Comment:
Untested, but I think we want something like this. The previous code was dealing with only two potential Node versions, but now we have more (and upcoming Meteor versions add even more):

```shell
	# Versions 1.8.x and below need Node 8.17.0
	if [[ "${version}" == 1.6* ]] || [[ "${version}" == 1.7* ]] || [[ "${version}" == 1.8* ]]; then
		node_version='8.17.0'

	# Versions 1.9 through 2.2.x need Node 12.22.1
	elif [[ "${version}" == 1.9* ]] || [[ "${version}" == 2.0* ]] || [[ "${version}" == 2.1* ]]; then
		node_version='12.22.1'

	# Versions >= 2.3 need Node 14.17.1
	else
		node_version='14.17.1'
	fi
```

Reply:
I modified the code like in the code you provided. Tests are passing properly https://github.com/chneau/meteor-base/actions/runs/1027745165